packages/app/lib/utils/index.js

```diff
@@ -31,6 +31,44 @@ class FirebaseUtilsModule extends FirebaseModule {
     return this.native.isRunningInTestLab;
   }
 
+  get playServicesAvailability() {
+    if (isIOS) {
+      return {
+        isAvailable: true,
+        status: 0,
+      };
+    }
+    return this.native.androidPlayServices;
+  }
+
+  getPlayServicesStatus() {
+    if (isIOS) {
+      return Promise.resolve(null);
+    }
+    return this.native.androidGetPlayServicesStatus();
+  }
+
+  promptForPlayServices() {
+    if (isIOS) {
+      return Promise.resolve(null);
+    }
+    return this.native.androidPromptForPlayServices();
+  }
+
+  makePlayServicesAvailable() {
+    if (isIOS) {
+      return Promise.resolve(null);
+    }
+    return this.native.androidMakePlayServicesAvailable();
+  }
+
+  resolutionForPlayServices() {
+    if (isIOS) {
+      return Promise.resolve(null);
```

Comment:
Your types now say these return undefined on iOS; updated below to match;

```suggestion
      return Promise.resolve();
    }
    return this.native.androidGetPlayServicesStatus();
  }

  promptForPlayServices() {
    if (isIOS) {
      return Promise.resolve();
    }
    return this.native.androidPromptForPlayServices();
  }

  makePlayServicesAvailable() {
    if (isIOS) {
      return Promise.resolve();
    }
    return this.native.androidMakePlayServicesAvailable();
  }

  resolutionForPlayServices() {
    if (isIOS) {
      return Promise.resolve();
```

Reply:
I'm confused, I thought my types for iOS were returning `null` 😅? If you prefer `undefined` I'll change accordingly 👍

Reply:
Types only mention void (undefined), e.g. `resolutionForPlayServices(): Promise<void>;`:

![image](https://user-images.githubusercontent.com/5347038/75469745-6fd66500-5987-11ea-99f0-25f960e274cf.png)

If you change this then the `@android` docs tag on each probably also needs updating

Reply:
Ah, I see. Cool, I'll update to  :`Promise<void | null>` for the types then 👍